src/components/items/Event.jsx

```diff
@@ -143,7 +143,7 @@ function Event(props) {
           {ishost ? null : (
             <NavItem>
               <NavLink
```

Comment:
this will need to be commented back in.

Reply:
I'll fix this

Reply:
Actually, I'm not sure what this is referring to, I didn't change this part.